wps/classes/wps.listener.php

```diff
@@ -201,6 +206,10 @@ protected function onmasteradminGetMenuContent($event)
             // add config page for projects restriction options
             $sectionAuth->childItems[] = new masterAdminMenuItem('wps_proj_restrict', \jLocale::get('wps~wps.ui.admin.menu.project_restriction'), jUrl::get('wps~restrictionsAdmin:show'), 160, 'wps_admin');
         }
+        if (jAcl2::check('wps.urlserver.manage')) {
```

Comment:
A right is not enough. We should hide this menu item when lizmap is configured to hide sensitive informations (on our saas platform, users are not authorized to change these parameters).

So here, you should call `if (\lizmap::getServices()->hideSensitiveProperties()) {....}` to know if you can show this item or not. You must call this same method in all actions of the controller, to forbid access to them.